recipes/gradle/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "gradle" %}
+{% set version = "5.3.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gradle/gradle/archive/v{{ version }}.tar.gz
+  sha256: f5bc12b30a44ffe79295c5a9a7ac6b1cfa52e99226cb40e6d18a05af91c73ea4
+  fn: gradle-{{ version }}.tar.gz
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+
+requirements:
+  host:
+    - coreutils >=8
+    - openjdk >=11
+  run:
+    - openjdk >=11
```

Comment:
can we remove this from run to let user decide if he wants to use the system java or install it via conda?

Reply:
It can be added to `test:` to make the tests work.